src/test/java/com/decide/DecideTest.java

```diff
@@ -367,15 +380,21 @@ public void LIC8TestInCircle(){
 
     @Test
     /**
-     * Tests that LIC8 does not find three points each separated by
-     * one point that can all be contained inside a circle of radius 2.
-     */
+     * Requirements: See `LIC8` documentation
+     * Contract:
+     *      Precondition: There exists a set of three points that are each
```

Comment:
This precondition doesn't really describe the precondition correctly: 
>There exists a set of three points that are each separated by one consecutive intervening point that can not be contained inside a circle of radius 2.

Should probably be switched to:
> No set of three points that are each separated by one consecutive intervening point can be contained inside a circle of radius 2.